test/xspec-bat.cmd

```diff
@@ -258,7 +258,17 @@ setlocal
 
     call :run ..\bin\xspec.bat ..\tutorial\escape-for-regex.xspec
     call :verify_retval 0
-    call :verify_line 19 "Report available at %PARENT_DIR_ABS%\tutorial\xspec\escape-for-regex-result.html"
+    call :verify_line 19 x "Report available at %PARENT_DIR_ABS%\tutorial\xspec\escape-for-regex-result.html"
+
+    call :end
+endlocal
+
+setlocal
+    call :begin "invoking xspec.sh that passes a non xs:boolean does not raise a warning #46"
+
+    call :run ..\bin\xspec.bat ..\test\xspec-46.xspec
+    call :verify_retval 0
+    call :verify_line 4 r "Testing with"

```

Comment:
This is fine now but I think it may break after #65 is merged into `master` (see notes in #65). I think this may need to be changed to:
```winbatch
call :verify_line 5 r "Testing with"
```